src/main/webapp/app/app.module.ts

```diff
@@ -28,7 +29,15 @@ import { LoginModule } from './login/login.module';
     // Set this to true to enable service worker (PWA)
     ServiceWorkerModule.register('ngsw-worker.js', { enabled: false }),
   ],
-  declarations: [MainComponent, NavbarComponent, ErrorComponent, PageRibbonComponent, ActiveMenuDirective, FooterComponent],
+  declarations: [
+    MainComponent,
+    NavbarComponent,
+    ErrorComponent,
+    PageRibbonComponent,
+    ActiveMenuDirective,
+    FooterComponent,
+    NavbarLogoutDialogComponent,
```

Comment:
Do we need to add this here? (only needed in the navbar component?)

Reply:
yes only in navbar component but the navbar component does not have a module to load the needed Components